src/index.html

```diff
@@ -1,13 +1,303 @@
+              <a href="#aboutUs" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compareBikes" class="menu__link">Compare Bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+
+            <a href="tel:+1 234 5555-55-55" class="menu__call menu__call--phone">+1 234 5555-55-55</a>
+            <a href="tel:+1 234 5555-55-55" class="menu__call menu__call--action">Book a test ride</a>
+          </ul>
+        </div>
+      </div>
+
+    </nav>
+
+    <main class="page__main">
+      <section class="page__section about" id="aboutUs">
+        <div class="container">
+          <div class="about__content">
+            <h2 class="page__section-title about__title">Move Free</h2>
+
+            <div class="about__desc">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </div>
+          </div>
+        </div>
+
+      </section>
+
+      <section class="page__section compare" id="compareBikes">
+        <div class="container">
+          <h2 class="page__section-title">Compare bikes</h2>
+          <div class="compare__wrap">
+            <div class="card compare__card">
+              <img
+                src="./images/bike-1.jpg"
+                alt="зображення велосипеду модель Sporty 4"
+                class="card__img"
+              >
+              <h3 class="card__title">Sporty 4</h3>
+              <p class="card__description">The iconic frame brought to a new performance height as a sporty, active ride.</p>
+              <span class="card__price">$ 2 590</span>
+            </div>
+
+            <div class="card compare__card">
+              <img
+                src="./images/bike-2.jpg"
+                alt="зображення велосипеду модель Ride in town ST"
+                class="card__img"
+              >
+              <h3 class="card__title">Ride in town ST</h3>
+              <p class="card__description">An open frame for an upright riding position as the most comfortable ride in town.</p>
+              <span class="card__price">$ 2 590</span>
+            </div>
+
+            <div class="card compare__card">
+              <img
+                src="./images/bike-3.jpg"
+                alt="зображення велосипеду модель Agile ride 3"
+                class="card__img"
+              >
+              <h3 class="card__title">Agile ride 3</h3>
+              <p class="card__description">The lightweight frame that has earned its street tread as a sleek, agile ride.</p>
+              <span class="card__price">$ 2 090</span>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section" id="details">
```

Comment:
U use just class page__section, I guess, it is sutible to add also class "details" - as it is main name of this block